docs/release-info/release-notes.md

```diff
@@ -83,6 +71,24 @@ Additionally, it helps in reducing load on the metadata store.
 
 [#16144](https://github.com/apache/druid/pull/16144)
 
+### Grouping on complex columns
+
+Druid now supports grouping on complex columns and nested arrays.
+This means that both native queries and the MSQ task engine can group on complex columns and nested arrays while returning results.
+
+Additionally, the MSQ task engine can rollup on supported complex dimensions, such as JSON columns, during ingestion.
```

Comment:
Since we removed the explicit sorting section. 
```suggestion
Additionally, the MSQ task engine can rollup and sort on the supported complex dimensions, such as JSON columns, during ingestion.  
```

Reply:
Updated